packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -149,10 +149,12 @@ export default function ClassifierContainer({
   }, [loaded, authClient])
 
   try {
-    if (!loaded) {
+    if (!loaded && !workflowData) {
```

Comment:
The logic here needs to be inverted: render the classifier if loaded and if the snapshot is loaded. Otherwise show a loading message.